### ISSUE TITLE
How to get values from ManyToManyRelationManager objects

### ISSUE THREAD

**Issue by asyncins** [tortoise/tortoise-orm#102]
```
# Model like this
class Permission(Model):
    id = IntField(pk=True)
    name = CharField(max_length=160)

    def __str__(self):
        return self.name


class Role(Model):
    id = IntField(pk=True)
    name = CharField(max_length=160)
    permissions = ManyToManyField('models.Permission', related_name='perm')
    create_time = DatetimeField(auto_now=True)

    def __str__(self):
        return self.name
```

```
# Handler like this

class RoleHandler(RequestHandler):
    async def get(self, *args, **kwargs):
        role = await Role.all()
        for r in role:
            print(i.permissions)
```
<tortoise.fields.ManyToManyRelationManager object at 0x7fcb78126668>
=====================================================
How to get values from ManyToManyRelationManager objects ?

**Comment by grigi** [comment]
Hi.

The documentation for relationships is lacking, and for that I apologise. I am working on a branch that fixed a few convenience issues for foreign relationships that could help.

The issue is that foreign relationships are lazy, and normally you should try to await the queryset.
e.g.:
```py
print(await i.permissions._query())
```
If you use #101 you should be able to just do:
```py
print(await i.permissions.all())
```

**Comment by grigi** [comment]
Sorry, `await i.permissions._query` without ()

**Comment by asyncins** [comment]
> Sorry, `await i.permissions._query` without ()

That's great

**Comment by asyncins** [comment]
> Hi.
> 
> The documentation for relationships is lacking, and for that I apologise. I am working on a branch that fixed a few convenience issues for foreign relationships that could help.
> 
> The issue is that foreign relationships are lazy, and normally you should try to await the queryset.
> e.g.:
> 
> ```python
> print(await i.permissions._query())
> ```
> 
> If you use #101 you should be able to just do:
> 
> ```python
> print(await i.permissions.all())
> ```

I use `i.permissions.all()` , running results are `<tortoise.fields.ManyToManyRelationManager object at 0x7fcb78126668>`

What should I do now?

**Comment by grigi** [comment]
Only use i.permissions.all() if you are using #101 (but I'm going to change behaviour for consistency)
What you can use in the mean time is:
```py
print(await i.permissions._query)
```
When I'm done with #101 I'll ask you to vet the changes.
I found that a lot of the confusion is due to the odd mix of sync and async interfaces, which confuses easily, and doesn't really work.
I'm changing it to always-lazy and always-async. We can add optimizations later on where it is safe.

**Comment by grigi** [comment]
Hi, can you please check if your issue has been resolved with v0.11.4?
The documentation has also been updated.
